suzieq/poller/services/interfaces.py

```diff
@@ -316,7 +316,7 @@ def _clean_junos_data(self, processed_data, raw_data):
                 elif entry['speed'].endswith('Gbps'):
                     entry['speed'] = int(entry['speed'].split('Gb')[0])*1000
                 elif entry['speed'] == 'Unlimited':
-                    entry['speed'] = 0
+                    entry['speed'] = -1
```

Comment:
This is another case where the speed should be 0. These interfaces are loopback interfaces, and are not included in interface assert check.